app/config/settings.py

```diff
@@ -1230,6 +1230,10 @@ def sentry_before_send(event, hint):
         "task": "grandchallenge.statistics.tasks.update_site_statistics_cache",
         "schedule": crontab(hour=5, minute=30),
     },
+    "send_onboarding_task_reminder_emails": {
+        "task": "grandchallenge.challenges.tasks.send_onboarding_task_reminder_emails",
+        "schedule": crontab(hour=6, minute=0),
```

Comment:
Do we want to send these emails every day? It might be too pushy. Maybe sending these only on Monday mornings is enough?

Reply:
It might be a bit pushy indeed. I do think it is good to send reminders and they are currently set to be sent when a task deadline is less then 72 hours away (crossing the weekend). That will still kinda work when we do it once a week.

What do you think about splitting the deadline alert and reminders? Have reminders daily but the actual deadline-crossed alert to once a week.

Reply:
This is something for @miriam-groeneveld to decide on I think.

Reply:
These are all emails right? So a reminder email when the deadline is less than 72 hours away, an alert email after the deadline has passed. If there are several tasks, people can get both reminders and deadline emails separately. Let's be cautious and send emails max once a week, Monday morning seems like a good time. That way, it's max 2 a week (1 reminder, 1 alert).

Reply:
How about a weekly email with "You have n tasks that are due sometime in the next week" and "You have n tasks that are overdue, please complete them as soon as possible, feel free to ask support for help etc"? Indicating the maximum overdue time might also be good, "you're 3 weeks late with this".

Reply:
I think the suggestion on having two emails a week would work nicely and the 'you are x weeks late' is a good push. 

I'll make the changes.

Reply:
Done. I think.

Reply:
The emails are changed but they're still sent every morning, right?

Reply:
Actually, they're not changed, @chrisvanrun did you forget to push?

Reply:
Eh, maybe? I thought I had pushed but also just quickly did a quickfix and pushed after.